client/ayon_core/plugins/publish/extract_burnin.py

```diff
@@ -116,11 +118,11 @@ def _get_burnins_per_representations(self, instance, src_burnin_defs):
             )
 
             # Filter output definition by `burnin` represetation key
-            repre_linked_burnins = {
-                name: output
-                for name, output in burnin_defs.items()
-                if name in repre_burnin_links
-            }
+            repre_linked_burnins = [
+                burnin_def
+                for burnin_def in burnin_defs.items()
```

Comment:
Are you sure this should still be `.items()`? Pretty sure this will currently fail.

Reply:
There should not be `.items()` 🙂 
```suggestion
                for burnin_def in burnin_defs
```